cognite/neat/rules/importers/_cdf_classic.py

```diff
@@ -0,0 +1,231 @@
+from cognite.neat.constants import CLASSIC_CDF_NAMESPACE
+from cognite.neat.issues import IssueList
+from cognite.neat.issues.errors import NeatValueError
+from cognite.neat.rules._shared import ReadRules
+from cognite.neat.rules.models import InformationInputRules
+from cognite.neat.rules.models.information import (
+    InformationInputClass,
+    InformationInputMetadata,
+    InformationInputProperty,
+)
+from cognite.neat.store import NeatGraphStore
+from cognite.neat.utils.rdf_ import remove_namespace_from_uri
+
+from ._base import BaseImporter
+
+
+class CDFClassicGraphImporter(BaseImporter[InformationInputRules]):
+    def __init__(self, store: NeatGraphStore) -> None:
+        self._store = store
+
+    def to_rules(self) -> ReadRules[InformationInputRules]:
+        issues = IssueList()
+        if not self._store.queries.has_namespace(CLASSIC_CDF_NAMESPACE):
+            issues.append(NeatValueError("The store does not contain the classic CDF namespace"))
+            return ReadRules(None, issues, read_context={})
+
+        rules = InformationInputRules(
+            metadata=InformationInputMetadata.load(self._default_metadata()),
+            properties=[],
+            classes=[],
+        )
+        self._add_source_system(rules)
+        self._add_dataset(rules)
+        self._add_asset(rules)
+        self._add_event(rules)
+        self._add_sequence(rules)
+        self._add_file(rules)
+
+        return ReadRules(rules, issues, read_context={})
+
+    def _add_source_system(self, rules: InformationInputRules) -> None:
+        types_ = self._store.queries.types_with_property(CLASSIC_CDF_NAMESPACE.source)
+        if not types_:
+            return
+        rules.classes.append(
+            InformationInputClass(
+                class_="SourceSystem",
+                parent="cdf_cdm:SourceSystem",
```

Comment:
Thinking is that we create classes and use the `parent` or `reference` to link it to the core model.